README.md

```diff
@@ -31,8 +32,6 @@ See `contributing.md` for ways to get started.
 
 Please adhere to this project's `code of conduct`.
 
-
 ## Authors
```

Comment:
Can you improve the Authors part ?
Like something creative displaying the profile for every contributor ?

Reply:
Okay, will do!

Reply:
@SauveJeanLuc are you alright with me using the AllContributors bot?

Reply:
No worries @kubahasek.
You can use it

Reply:
@SauveJeanLuc I'd need you to install it though. I don't have the permissions to do it for this repo. <https://allcontributors.org/docs/en/bot/installation>, here is the link to it.

Reply:
@kubahasek.
I've installed it.